meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab

```diff
@@ -1,4 +1,4 @@
-# Eclipse Leda - fstab for qemuarm RAUC redundant boot setup
+# Eclipse Leda - fstab for qemuarm64 RAUC redundant boot setup
 
```

Comment:
qemuarm64 -> qemuarm

Reply:
Fixed in latest version in file: https://github.com/SoftwareDefinedVehicle/meta-leda-fork/blob/make-qemuarm32-work/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab#L1